react/store/createReturnRequest/components/ItemsDetails.tsx

```diff
@@ -153,6 +171,103 @@ export const ItemsDetails = (props: Props) => {
+                <FormattedMessage id="store/return-app.return-order-details.table-header.quantity" />
+                :
+              </span>
+              <span className={`${handlesPhone.quantityValue}`}>
+                {quantity}
+              </span>
+            </div>
+            <div
+              className={`${handlesPhone.availableToReturnWrapper} flex flex-row mb4`}
+            >
+              <span className={`${handlesPhone.availableToReturnKey} mr2`}>
+                <FormattedMessage id="store/return-app.return-order-details.table-header.available-to-return" />
+                :
+              </span>
+              <span className={`${handlesPhone.availableToReturnValue}`}>
+                {availableToReturn}
+              </span>
+            </div>
+            {isExcluded ? (
+              <CustomMessage
+                status="warning"
+                message={
+                  <FormattedMessage id="store/return-app.return-item-details.excluded-items.warning" />
+                }
+              />
+            ) : null}
+            <div className={`${handlesPhone.quantitySelectorWrapper} mb4`}>
+              <NumericStepper
+                size="small"
+                maxValue={availableToReturn}
+                value={currentItem?.quantity ?? 0}
+                onChange={(e: { value: number }) =>
+                  handleQuantityChange(e.value)
+                }
+              />
+            </div>
+            <div className={`${handlesPhone.reasonWrapper} mb4`}>
+              <RenderReasonDropdown
+                isExcluded={isExcluded}
+                reason={currentItem?.returnReason?.reason ?? ''}
+                otherReason={currentItem?.returnReason?.otherReason ?? ''}
+                onReasonChange={handleReasonChange}
+                creationDate={creationDate}
+              />
+              {reasonError && reasonErrorEmptyValue ? (
+                <CustomMessage
+                  status="error"
+                  message={
+                    <FormattedMessage id="store/return-app.return-item-details.dropdown-reason.error" />
+                  }
+                />
+              ) : null}
+            </div>
+            {!enableSelectItemCondition ? null : (
+              <div className={`${handlesPhone.conditionWrapper}`}>
+                <RenderConditionDropdown
+                  isExcluded={isExcluded}
+                  condition={currentItem?.condition ?? ''}
+                  onConditionChange={handleConditionChange}
+                />
+                {conditionError ? (
+                  <CustomMessage
+                    status="error"
+                    message={
+                      <FormattedMessage id="store/return-app.return-item-details.dropdown-condition.error" />
+                    }
+                  />
+                ) : null}
+              </div>
+            )}
+          </div>
+        </Box>
+      </div>
+    )
+  }
+
```

Comment:
Since we are returning the Card if the user is using a mobile phone, can we remove the references from the table (lines 161, 190, 233)